melodic/distribution.yaml

```diff
@@ -1263,6 +1263,21 @@ repositories:
       type: git
       url: https://github.com/whoenig/crazyflie_ros.git
       version: master
+  crop_hull_filter:
+    doc:
+      type: git
+      url: https://github.com/OUXT-Polaris/crop_hull_filter.git
```

Comment:
This repository declares the license as Apache 2.0 in the package.xml (good!), but doesn't have a LICENSE file (bad).  Please add in the LICENSE file to the source repository.  It is also customary to add the shortened version of the license to the top of the source and include files, but I won't insist on that.